diagnostic.md

```diff
@@ -92,7 +132,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+The div labeled 'specialDiv' will have a text color of yellow and a font family of sans-serif.
```

Comment:
There is inline styling the takes precedent over the stylesheet specifying that the font-family will be cursive